generators/client/templates/angular/src/main/webapp/app/shared/tracker/_tracker.service.ts

```diff
@@ -11,6 +10,9 @@ import { CSRFService } from '../auth/csrf.service';
 import { AuthServerProvider } from '../auth/auth-jwt.service';
 <%_ } _%>
 
+import SockJS = require('sockjs-client');
```

Comment:
thats strange as per the examples on this page https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import it should be possible to import functions as `import { SockJs } from 'sockjs-client'` or `import SockJs  from 'sockjs-client'`